<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>listarezerwacji</class>
 <widget class="QWidget" name="listarezerwacji">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell Condensed</family>
     <pointsize>60</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Spanie1</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>591</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>171</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Podaj ID rezerwacji:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Wprowadź nową datę rezerwacji:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Od</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="odDaty"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Do</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="doDaty"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Zapisz zmiany</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Zamelduj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Wymelduj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Odwołaj rezerwację</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>Wróć</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
